c2corg_api/views/__init__.py

```diff
@@ -1,6 +1,32 @@
 import collections
 import datetime
 from colander import null
+from pyramid.httpexceptions import HTTPError, HTTPNotFound
+from pyramid.view import view_config
+from cornice import Errors
+from cornice.util import json_error
+
+
+@view_config(context=HTTPNotFound)
+@view_config(context=HTTPError)
```

Comment:
Does this means that when an HTTPNotFound or HTTPError is detected, http_error_handler() will automatically be triggered to add the JSON error output as the response?